### ISSUE TITLE
sort by date breaks for some date formats

### ISSUE THREAD

**Issue by mhinsch** [mickstar/Zoo-For-Zotero#125]
If the `date` field contains the month and the year, e.g. `12/2014`, it seems the 12 is interpreted as the (beginning of?) the year.

**Comment by mickstar** [comment]
Yes it's very difficult to sort dates in zotero because the dates are so inconsistent, I will add some logic to try to pull out the year from the date.

Is MM/YYYY used in your library?

**Comment by mhinsch** [comment]
Yes it is, but not intentionally. For some publication types zotero seems to think that sub-year information is relevant. Personally I've never encountered a situation where I need that information so I try to remove anything but the year whenever I notice, but maybe there are people who actually do need it.

**Comment by mickstar** [comment]
Sure, I will add some regex to pull out 4 digit numbers and make that the
most important number for sorting
